app/Models/EntryDAOPGSQL.php

```diff
@@ -23,6 +23,32 @@ public static function sqlIgnoreConflict(string $sql): string {
 		return rtrim($sql, ' ;') . ' ON CONFLICT DO NOTHING';
 	}
 
+	#[\Override]
+	protected static function sqlRegex(string $expression, string $regex, array &$values): string {
+		$matches = static::regexToSql($regex);
+		if (isset($matches['pattern'])) {
+			$matchType = $matches['matchType'] ?? '';
+			if (str_contains($matchType, 'm')) {
+				// newline-sensitive matching
+				$matches['pattern'] = '(?m)' . $matches['pattern'];
```

Comment:
It looks like metasyntax could be specified for Postgres, but only if you don't use `/../m` and include the `m` in the set of metasyntax options manually if you need it.  As a power user it's good to know, but I'm uncertain whether or not it might be worth mentioning in documentation somewhere.